cpg_infra/billing_aggregator/aggregate/utils.py

```diff
@@ -513,24 +513,46 @@ async def get_jobs_for_batch(
 async def get_batch_by_id(
```

Comment:
Ehhh, it's not a big deal I guess, but I see lots of individual calls to this. I wonder if it's worth adding a cache decorator to this function?

Reply:
It is called with different params, so do not think it is wort it. It would be called max 2x per run: 
if loaded by batch id (max 2x), if by date then only once.